manuscript/manuscript.md

```diff
@@ -70,6 +70,14 @@ organization:
 
 # Introduction {-}
 
+Data is the central currency of science.  However the nature of scientific data has changed dramatically with the rapid pace of technology, leading to an increasing heterogeneity. Improvements in high throughput DNA sequencing and sustained institutional support for large sensor networks [@Reid2014, @Hampton2013] have lead to the creation of massive quanities of data. At the same time collaboration between researchers is becoming increasingly common [@Adams2012] with increased coordination between researchers collecting data [@Fraser2013]. These changes mean that data is more heterogenous than ever before in both size and complexity. It ranges from petabytes of information stored in hadoop clusters to Excel spreadsheets and lab notebooks on shelves.  
```

Comment:
In the second sentence, heterogeneity of what? Data formats? Data use cases? Researchers' technical backgrounds? Also I think perhaps hadoop is something that should probably be explained when it is first mentioned since I don't think it's a common tool  for most biologists. We could say 'computer clusters in the cloud' instead?